### ISSUE TITLE
Move away from google calendar

### ISSUE THREAD

**Issue by rejon** [Fabricatorz/stl.ist#1]

**Comment by bebopkid** [comment]
Not suggesting we move to this format (for a few reasons), but for reference, here is the most successful event listing site in NY: http://www.garysguide.com/

**Comment by rejon** [comment]
Yeah I have a path out of this in mind. There is a csv jekyll based way to do that I think makes next sense but then someone has to hand enter, but gives less hacky interface but then would require github account to add/edit until could move to another solution...

could also move to drupal but would prefer go towards more control platfom as use ramps up...

Could also add disqus or other social sharingtk events

even better if events originated from here rather than follow on so have swarm effect

Once control code path, then also accept text in from various sources and convert to requests, like from emails or entered on the site.

Main thing is get this flowing and people submitting events and have that issue first before more dev hours

Jon

**Comment by rejon** [comment]
also can style this google calendar just did not put in the time, probablt easier to swap out tech and change jekyll version on next version
